pages/dkp/kommander/2.1/release-notes/2.1.3/index.md

```diff
@@ -0,0 +1,222 @@
+---
+layout: layout.pug
+navigationTitle: Kommander 2.1.3 Release Notes
+title: Kommander 2.1.3 Release Notes
+menuWeight: 40
+excerpt: View release-specific information for Kommander 2.1.3
+enterprise: false
+beta: false
+---
+**D2iQ&reg; Kommander&reg; version 2.1.3 was released on August, 25 2022.**
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Kommander[/button]
+
+To get started with Kommander, [download](../../download/) and [install](../../install/) the latest version of Kommander.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Kommander.</p>
+
+## Release summary
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Kommander.
+
+## Fixes and Improvements
+
+### Upgrade Konvoy 1.8 with Multiple Availability Zones (Incident 9448)
+
+With this patch release, multiple Availability Zones are now supported when upgrading from Konvoy v1.8.
+
+## Component updates
```

Comment:
Did any of these change? Do we want to go to doing table format?

Reply:
I figured easier to leave as-is for the point release, but we could change to the table.

Reply:
@dkoshkin @gracedo - are there any component changes/updates since 2.1.2?

Reply:
just knative for us (updated and committed)